Source/Core/Core/Core.cpp

```diff
@@ -668,7 +674,7 @@ State GetState()
 
 void WaitUntilDoneBooting()
 {
-  if (s_is_booting.IsSet() || !s_hardware_initialized)
+  if (s_is_booting.IsSet() || !s_hardware_initialized.IsSet())
```

Comment:
You could make this a little bit neater by calling `IsBooted`.

Reply:
Are `s_is_booting` and `!s_is_stopping` functionally equivalent? I'll need to analyze it closely but I did not touch this on an assumption they are not.

Reply:
If I'm understanding it correctly, there will be a difference in behavior for the case where this is called while we are stopping. But the only code that calls this doesn't run when we are stopping or already have stopped, so this difference shouldn't affect anything.

Reply:
Good point - I'll try to eyeball this later.